tests/phpunit/Helpers/WC_Subscriptions_Product.php

```
@@ -43,9 +61,13 @@ public static function set_trial_length( int $result ): void {
 	 * Determine if a product is a subscription.
 	 *
+	 * @param \WC_Product|null $product The product to check if it is a subscription.
 	 * @return bool
 	 */
-	public static function is_subscription(): bool {
+	public static function is_subscription( $product = null ): bool {
+		if ( ! empty( self::$subscription_product_ids ) && $product instanceof \WC_Product ) {
+			return in_array( $product->get_id(), self::$subscription_product_ids, true );
+		}
 		return self::$is_subscription_result;
 	}
```

PHPMD (2.15.0)

[error] The method is_subscription is not named in camelCase. (undefined)

(CamelCaseMethodName)